packages/stog/stog.0.15.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "zoggy@bat8.org"
+
+authors: ["Maxence Guesdon"]
+homepage: "http://zoggy.github.io/stog/"
+license: "GNU Public License version 3"
+doc: ["http://zoggy.github.io/stog/doc.html"]
+dev-repo: "https://github.com/zoggy/stog.git"
+bug-reports: "https://github.com/zoggy/stog/issues"
+tags: ["publication" "xml" "documentation" "blog" "web" "website"]
+build: [
+  ["./configure" "--prefix" prefix]
+  [make "all"]
+]
+install: [
+  [make "install-lib" "install-share"]
+]
+remove: [["ocamlfind" "remove" "stog"]]
+depends: [
+  "ocamlfind"
+  "xmlm" {>= "1.1"}
+  "xtmpl" {>= "0.12"}
+  "ocf" {>= "0.3"}
+  "ocamlnet" {>= "3.6"}
+  "higlo" { >= "0.4" }
+  "ppx_blob" {>= "0.1"}
+]
+depopts: [
+  "js_of_ocaml" { >= "2.5"}
+  "xmldiff" { >= "0.5.0" }
+  "lwt" { >= "2.4.5" }
+  "websocket" { >= "2.1" }
+  "ojs-base" { >= "0.3" }
+  "sha" { >= "1.9" }
+]
```

Comment:
These are not conflicts, i.e. having previous versions installed won't prevent stog from being installed, but the server won't be compiled and built. I found no way to indicate this. depopts should accept some kind of conditions. In this case, the given packages and versions must all be present for the server to be compiled and built. But by now, any change in one of the depopts will trigger a reinstallation of stog.